python-test2.py

```diff
@@ -1,61 +1,83 @@
 import random
 
 class InventoryItem:
+    """Represents an item in inventory with name, quantity, and price."""
+
     def __init__(self, name, quantity, price):
+        if quantity < 0 or price < 0:
```

Comment:
@SeineSailor tell me if this change safe in handling race condition, or will it introduce any type mismatch, or out of range errors?